Gems/AzQtComponentsForPython/Code/CMakeLists.txt

```diff
@@ -0,0 +1,146 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+#
+
+set(GEM_NAME AzQtComponentsForPython)
+
+o3de_pal_dir(pal_dir ${CMAKE_CURRENT_LIST_DIR}/Platform/${PAL_PLATFORM_NAME} "${gem_restricted_path}" "${gem_path}" "${gem_parent_relative_path}")
+
+include(${pal_dir}/PAL_${PAL_PLATFORM_NAME_LOWERCASE}.cmake)
+
+if (PAL_TRAIT_AZQTCOMPONENTSFORPYTHON_SUPPORTED)
+    # The name of the Python import module we're generating.
+    set(BINDINGS_LIBRARY "azqtpyside")
+
+    # Header file containing all the types/classes we're exposing to Python.
+    set(wrapped_header ${CMAKE_CURRENT_LIST_DIR}/bindings_azqtcomponents.h)
+
+    # Typesystem xml.
+    set(typesystem_file ${CMAKE_CURRENT_LIST_DIR}/bindings_azqtcomponents.xml)
+
+    include(${CMAKE_CURRENT_LIST_DIR}/wrapped_files.cmake)
+    
+    # Allow AUTOMOC/AUTOUIC on generated files.
+    if(POLICY CMP0071)
+      cmake_policy(SET CMP0071 NEW)
+    endif()
+
+    project(AzQtComponent_Bindings LANGUAGES CXX)
+    set(CMAKE_AUTOMOC ON)
+    find_package(Qt5 5.12 REQUIRED COMPONENTS Core Gui Widgets)
+                 
+    # The library we're creating bindings for.
+    set(AZQTCOMPONENTS_LIBRARY "AzQtComponents")
+
+    # Check we have a valid version of Python.
+    if(NOT python_interpreter)
+        find_program(python_interpreter "python")
+    endif()
+
+    if(NOT python_interpreter)
+        message(FATAL_ERROR "Error: Failed to locate Python installation.")
+    endif()
```

Comment:
We should use the python downloaded with O3DE.
That is accessible through the [LY_PYTHON_CMD](https://github.com/o3de/o3de/blob/development/cmake/Platform/Windows/PAL_windows.cmake#L44) variable.

Reply:
Good point, I'll remove that check.